src/controllers/tracker.controller.js

```diff
@@ -0,0 +1,13 @@
+import commandMap from "./commands/commandMap.js";
+
+export function trackerController(bot) {
```

Comment:
nit: given it's already under `controllers`, can this just be named `tracker`?

Reply:
sure. could go either way.

Reply:
fixed